generator/src/test/java/com/scottlogic/deg/generator/violations/filters/ConstraintTypeViolationFilterTest.java

```diff
@@ -0,0 +1,54 @@
+package com.scottlogic.deg.generator.violations.filters;
+
+import com.scottlogic.deg.generator.constraints.Constraint;
+import com.scottlogic.deg.generator.constraints.atomic.IsGreaterThanConstantConstraint;
+import com.scottlogic.deg.generator.constraints.atomic.IsInSetConstraint;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.Collections;
+
+import static org.hamcrest.CoreMatchers.is;
+
+/**
+ * Defines tests for the ConstraintTypeViolationFilter
+ */
+public class ConstraintTypeViolationFilterTest {
```

Comment:
Should this be pluralised? i.e. `Tests`
https://github.com/ScottLogic/datahelix/wiki/Testing-strategy#junit-jupiter

Reply:
> Ensure fixture class is named <production-class-name>Tests, like in example above

Reply:
It should, good spot. I think my IntelliJ generates singuluar by default - I caught it on the other one